nf_core/modules/lint/meta_yml.py

```diff
@@ -195,6 +199,26 @@ def meta_yml(module_lint_object: ComponentLint, module: NFCoreComponent, allow_m
                     )
                 )
 
+        # Check that ext.args are listed in the `extra_args` section
+        expected_extra_args = module.get_ext_args_from_main_nf()
+        extra_args_yml = sorted(meta_yaml_content.get("extra_args", {}).keys())
```

Comment:
It is called `args` in the `meta.yml`, we should either change it here or in the template
```suggestion
        extra_args_yml = sorted(meta_yaml_content.get("args", {}).keys())
```

Reply:
Oops, I started with`args` but I felt that could be somewhat ambiguous and also potentially confusing when the section looks like:

```yml
args:
  - args:
``` 

Personal preference would be to use `extra_args` for the section title?

Reply:
sounds good 👍